### PR TITLE
156-VOMongoRepository-InstanceVariable-host--port-are-unuse

### LINKED ISSUE #156: VOMongoRepository InstanceVariable host & port are unuse
I think we should remove those 2 IVs and add methods to get those data from the IV *mongoUrls*.

What do you think ?

### PR DESCRIPTION
**Pull request by CafeKrem**
add method to get ports and hosts from mongoUrls fix #156

### CODE REVIEW

File: mc/Voyage-Mongo-Core.package/VOMongoRepository.class/instance/hosts.st [hunk 1]
```diff
@@ -0,0 +1,3 @@
+as yet unclassified
```
Comment (noha) [inline mc/Voyage-Mongo-Core.package/VOMongoRepository.class/instance/hosts.st:1]: Can you categorize the methods in "accessing"

File: mc/Voyage-Mongo-Core.package/VOMongoRepository.class/instance/hosts.st [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+hosts
```
Comment (noha) [inline mc/Voyage-Mongo-Core.package/VOMongoRepository.class/instance/hosts.st:2]: Is there a reason for the accessors?

Comment (CafeKrem) [reply mc/Voyage-Mongo-Core.package/VOMongoRepository.class/instance/hosts.st:2]: no you right ,  we should let this choice to the user

Comment (CafeKrem) [reply mc/Voyage-Mongo-Core.package/VOMongoRepository.class/instance/hosts.st:2]: I will remove it

**Pull request merged**